layouts/shortcodes/logos.html

```diff
@@ -2,36 +2,48 @@
   <div class="mt-4 col col-md-12 col-12">
     <h2 class="mb-3">Parsec is an open source collaborative project</h2>
   </div>
-  <div class="mt-4 col col-md-3 col-6">
+  <div class="mt-4 col col-md-3 col-6" style="text-align:center">
```

Comment:
Consider defining the `text-align:center` styling using SASS rather than hard coding the styling on each element.

Reply:
I think that might actually be what [line 1](https://github.com/parallaxsecond/parsec-website/blob/4c7f59c70fbda0ab8c14bff0042afeb535ba4246/layouts/shortcodes/logos.html#L1) may be trying to do...